### PR TITLE
media-fonts/intel-one-mono: new package, add 1.3.0

### PR DESCRIPTION
**Pull request by Kangie**

### CODE REVIEW

**Comment by Kangie** [comment]
Looks and works fine:

image

I considered building the font from source, but upstream recommend using [RoboFont](https://robofont.com/), a non-free macOS-only font editor.

While it's theoretically possible to use any font builder that understands the UFO (v3[?]) format, I'm unwilling to put in the effort to work out how to automate that to provide the same output as Intel's dev environment for every update.

File: media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+S="${WORKDIR}/otf"
+
+BDEPEND="
+	app-arch/unzip
+"
+
+src_compile() { :; }
+
+src_install() {
+	FONT_SUFFIX="otf"
```
Comment (thesamesam) [inline media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild:27]: Can you set this part pre-inherit, then we can drop src_install entirely?

File: media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+SRC_URI="https://github.com/intel/intel-one-mono/releases/download/V${PV}/otf.zip -> ${P}.zip"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="binchecks strip"
+
+S="${WORKDIR}/otf"
```
Comment (thesamesam) [inline media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild:18]: Put this directly under SRC_URI

File: media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
```
Comment (thesamesam) [inline media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild:4]: Newline after EAPI

File: media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
```
Comment (thesamesam) [inline media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild:1]:
```suggestion
# Copyright 2023 Gentoo Authors
```

**Review by thesamesam — COMMENTED** [review]
Please add the rationale wrt no source in the commit message.

**Comment by Kangie** [comment]
Actioned.

When this is merged you'll need to tidy up this:
https://gitweb.gentoo.org/repo/proj/guru.git/tree/media-fonts/intel-one-mono/intel-one-mono-1.3.0.ebuild

I don't see a point in bringing in the `ttf` fonts here; `otf` should cover that use case in 2023.

**Pull request closed without merging**